### PR TITLE
Remove old and ignored integration tests

### PR DESCRIPTION
**Pull request by gimaker**
This lets us remove some arcane conditional logic in SystemTestBase
which results in different behaviour depending on the name of the
class.

### CODE REVIEW

**Comment by gimaker** [comment]
@rculbertson @rohansingh @mattnworb @davidxia

File: helios-system-tests/src/main/java/com/spotify/helios/system/SystemTestBase.java [hunk 1]
```diff
@@ -211,18 +207,8 @@ public void baseSetup() throws Exception {
     masterAdminPort = temporaryPorts.localPort("helios master admin");
 
     String className = getClass().getName();
```
Comment (davidxia) [inline helios-system-tests/src/main/java/com/spotify/helios/system/SystemTestBase.java]: `className` is now not needed

**Comment by davidxia** [comment]
We can also remove from SystemTestBase `masterAdminEndpoint()`. `startJob()`, `deregisterHost()`, and `readLogFully()` are also unused.

**Comment by rculbertson** [comment]
+1 to @davidxia's comments, otherwise :+1:

File: helios-system-tests/src/main/java/com/spotify/helios/system/SystemTestBase.java (hunk 1 above)
Comment (gimaker) [inline helios-system-tests/src/main/java/com/spotify/helios/system/SystemTestBase.java]: Done.

**Comment by gimaker** [comment]
I'll leave the removal of the unrelated but unused `masterAdminEndpoint()`. `startJob()`, `deregisterHost()`, and `readLogFully()` to another PR.

**Comment by davidxia** [comment]
:+1:

**Comment by mattnworb** [comment]
Is there a short answer as to why all these tests were disabled since 084adaaa8de1c428f312347ba81335b0784a1b0e ?

**Comment by mattnworb** [comment]
:+1: to cleanup though - it's too bad codecov doesn't count reduced complexity in the tests themselves

**Pull request merged**